src/components/general/Navbar/index.js

```diff
@@ -354,19 +326,52 @@ export default class Navbar extends React.PureComponent {
                 </div>
                 <NavDrop
                     className={styles.userMenu}
-                    links={this.validDropLinks}
+                    links={validDropLinks}
                     adminPanelLink={adminPanelLink}
-                    onLogout={this.handleLogoutClick}
+                    onLogout={handleLogoutClick}
                 />
                 <Confirm
                     show={showLogoutConfirm}
-                    onClose={this.handleLogoutModalClose}
+                    onClose={handleLogoutModalClose}
                 >
                     <p>
                         {_ts('components.navbar', 'logoutConfirmationText')}
                     </p>
                 </Confirm>
-            </nav>
-        );
-    }
+            </div>
+            {showSubNavbar[currentPath] && (
+                <div className={styles.subNavbar}>
+                    <div className={styles.projectSelectInputWrapper}>
+                        <Cloak
```

Comment:
Does showing a sub navbar add this project jump by default?

Reply:
Yes